File: app/src/main/java/com/incite/moviequiz/domain/model/Film.java
package com.incite.moviequiz.domain.model;

import android.graphics.drawable.Drawable;

public class Film {
    int drawableID;

    boolean isPassed = false;

    public void setPassed(boolean passed) {
        isPassed = passed;
    }

    public boolean isPassed() {
        return isPassed;
    }

    public int getDrawableID() {
        return drawableID;
    }

    public String getAnswer() {
        return answer;
    }

    String answer;

    public Film(String answer, int drawableID) {
        this.drawableID = drawableID;
        this.answer = answer;
    }
}

File: app/src/main/java/com/incite/moviequiz/domain/model/Film.kt
package com.incite.moviequiz.domain.model

import com.incite.moviequiz.data.entity.FilmEntity

data class Film(

    val id: Int,
    val pack_id: Int = 0,
    var name: String = "",
    var image: String = "",
    var completed: Boolean = false
) {
    fun toFilmEntity(): FilmEntity {
        return FilmEntity(
            id = id,
            pack_id = pack_id,
            name = name,
            image = image,
            completed = completed,
        )
    }

}
